python/hail/methods/qc.py

```diff
@@ -169,3 +169,512 @@ def variant_qc(dataset, name='variant_qc'):
+            consequence_terms: Array[String],
+            distance: Int,
+            domains: Array[Struct{
+              db: String
+              name: String
+            }],
+            exon: String,
+            gene_id: String,
+            gene_pheno: Int,
+            gene_symbol: String,
+            gene_symbol_source: String,
+            hgnc_id: String,
+            hgvsc: String,
+            hgvsp: String,
+            hgvs_offset: Int,
+            impact: String,
+            intron: String,
+            lof: String,
+            lof_flags: String,
+            lof_filter: String,
+            lof_info: String,
+            minimised: Int,
+            polyphen_prediction: String,
+            polyphen_score: Double,
+            protein_end: Int,
+            protein_start: Int,
+            protein_id: String,
+            sift_prediction: String,
+            sift_score: Double,
+            strand: Int,
+            swissprot: String,
+            transcript_id: String,
+            trembl: String,
+            uniparc: String,
+            variant_allele: String
+          }],
+          variant_class: String
+        }
+
+    Parameters
+    ----------
+    dataset : :class:`.MatrixTable`
+        Dataset.
+    config : :obj:`str`
+        Path to VEP configuration file.
+    block_size : :obj:`int`
+        Number of rows to process per VEP invocation.
+    name : :obj:`str`
+        Name for resulting row field.
+    csq : :obj:`bool`
+        If ``True``, annotates VCF CSQ field as a String.
+        If ``False``, annotates with the full nested struct schema
+
+    Returns
+    -------
+    :class:`.MatrixTable`
+        Dataset with new row-indexed field `name` containing VEP annotations.
+    """
+
+    return MatrixTable(Env.hail().methods.VEP.apply(dataset._jvds, config, 'va.`{}`'.format(name), csq, block_size))
+
+
+@handle_py4j
+@typecheck_method(dataset=MatrixTable,
+                  config=strlike,
+                  block_size=integral,
+                  name=strlike)
+def nirvana(dataset, config, block_size=500000, name='nirvana'):
+    """Annotate variants using `Nirvana <https://github.com/Illumina/Nirvana>`_.
+
+    .. include:: ../_templates/experimental.rst
+
+    .. include:: ../_templates/req_tvariant.rst
+
+    :py:meth:`~hail.methods.nirvana` runs `Nirvana
+    <https://github.com/Illumina/Nirvana>`_ on the current dataset and adds a
+    new row field in the location specified by `name`.
+
+    Examples
+    --------
+
+    Add Nirvana annotations to the dataset:
+
+    >>> result = methods.nirvana(dataset, "data/nirvana.properties") # doctest: +SKIP
+
+    Notes
+    -----
+
+    ***Configuration***
+
+    :py:meth:`~hail.VariantDataset.nirvana` requires a configuration file. The
+    format is a `.properties file
+    <https://en.wikipedia.org/wiki/.properties>`__, where each line defines a
+    property as a key-value pair of the form `key = value`. ``nirvana`` supports
```

Comment:
I prefer an example from a plain text file (`key = value`) to use double-backticks.